src/Support_Endpoints/Core.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Core settings listing endpoint for Support Hub
+ */
+
+namespace Avidly\Support_Client\Support_Endpoints;
+
+class Core extends Support_Endpoint {
+	public function __construct() {
+		$this->route = '/core/';
+		$this->register();
```

Comment:
The register call should be elsewhere, or called outside the class. The purpose of the constructor is to initialize the class and not to execute code.